ext/opcache/config.m4

```diff
@@ -18,6 +18,13 @@ PHP_ARG_ENABLE([opcache-jit],
   [yes],
   [no])
 
+PHP_ARG_ENABLE([opcache-instruments],
+  [whether to enable Instruments support],
+  [AS_HELP_STRING([--enable-opcache-instruments],
+    [Enable macOS JIT performance instruments])],
+  [yes],
+  [no])
```

Comment:
I think that we could compile this by default, since it's opt-in at runtime

Reply:
Fair enough :)